app/init.php

```diff
@@ -398,10 +418,10 @@ function($value) {
             $request->getCookie(Auth::$cookieName.'_legacy', '')));// Get fallback session from old clients (no SameSite support)
 
     // Get fallback session from clients who block 3rd-party cookies
-    $response->addHeader('X-Debug-Fallback', 'false');
+    if($response) $response->addHeader('X-Debug-Fallback', 'false');
```

Comment:
Let's add a comment explaining this condition for future reference.

Reply:
Actually not sure anymore where this change comes from.

Do you remember? https://github.com/appwrite/appwrite/commit/619781a4ad567b6def0add82e8a9fcf9cb48f83a

Reply:
Maybe for realtime, workers or CLI where we use this init, but don't have a request